src/state/farms/fetchFarmsPrices.ts

```diff
@@ -85,7 +85,7 @@ const getFarmQuoteTokenPrice = (
   return BIG_ZERO
 }
 
-const fetchFarmsPrices = async (farms: SerializedFarm[]) => {
+const fetchFarmsPrices = (farms: SerializedFarm[]) => {
```

Comment:
Should we rename this to `getFarmPrices`? Since its not really fetching anything, just processing.

Reply:
Done